src/pages/index.js

```diff
@@ -116,9 +117,18 @@ export default () => {
           />
         </div>
       </Banner>
-      <div id='about' style={{ minHeight: '100vh' }}>
+      <div id='about'>
         <About />
+        <div
+          id='why-mun'
+          style={{
+            background: theme.palette.background.pinkish,
+          }}
```

Comment:
apply this styling to outer div of about section

Reply:
done